autogen.sh

```diff
@@ -42,10 +40,9 @@ case "$1" in
         ;;
     "sysdeps")
             DEPS_LIST=$(list_build_dependencies)
-
             if [ "$2" == "--install" ]; then
                 set -x verbose
-                sudo dnf install --setopt=strict=0 $DEPS_LIST
+                eval sudo dnf install --setopt=strict=0 $DEPS_LIST
```

Comment:
Hmm, why eval?

Reply:
That's tricky :)  When DEPS_LIST="foo 'bar >= 1.0'" then `sudo dnf install $DEPS_LIST` expands to `sudo dnf install foo 'bar' '>=' '1.0'` but if you eval it first (before calling sudo) you will get `sudo dnf install foo 'bar >= 1.0'` and that is what we want.